PURL-TYPES.rst

```diff
@@ -311,6 +311,22 @@ huggingface
       pkg:huggingface/microsoft/deberta-v3-base@559062ad13d311b87b2c455e67dcd5f1c8f65111?repository_url=https://hub-ci.huggingface.co
 
 
+luarocks
+--------
+``luarocks`` for Lua packages installed with LuaRocks:
+
+- ``namespace``: The user manifest under which the package is registered. If not given, the root manifest is assumed.
+- ``name``: The LuaRocks package name.
+- ``version``: The full LuaRocks package version, including module version and rockspec revision.
```

Comment:
Is the version case sensitive or insensitive? Please specify which. Additionally, is the version required?